addons/packages/contour/1.21.0/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: imgpkg.carvel.dev/v1alpha1
+images:
+- annotations:
+    kbld.carvel.dev/id: docker.io/envoyproxy/envoy:v1.22.0
+    kbld.carvel.dev/origins: |
+      - resolved:
+          tag: v1.22.0
+          url: docker.io/envoyproxy/envoy:v1.22.0
+      - preresolved:
+          url: index.docker.io/envoyproxy/envoy@sha256:e9ad4cde73ffb01402c4e3fb6ce1a764d9e84650230135b88495f30904a378fc
+  image: index.docker.io/envoyproxy/envoy@sha256:e9ad4cde73ffb01402c4e3fb6ce1a764d9e84650230135b88495f30904a378fc
+- annotations:
+    kbld.carvel.dev/id: ghcr.io/projectcontour/contour:v1.21.0
+    kbld.carvel.dev/origins: |
+      - resolved:
+          tag: v1.21.0
+          url: ghcr.io/projectcontour/contour:v1.21.0
+      - preresolved:
+          url: ghcr.io/projectcontour/contour@sha256:6e6c6cbe03eba0df8987da4cac2712845f66c496185383d8768f19b40be26125
+  image: ghcr.io/projectcontour/contour@sha256:6e6c6cbe03eba0df8987da4cac2712845f66c496185383d8768f19b40be26125
```

Comment:
This doesn't look to have changed from the 1.20.1 package, the `make lock-package-images` should update this but if it's not, need to dig into what's going wrong